packages/cli/src/commands/studio/deploy.ts

```
@@ -15,6 +15,8 @@ import { getToken, getCurrentOrgId } from '../auth/credentials.js';
 import { preflightBuildOutput, printPreflightIssues } from '../deploy-preflight.js';
 import { fetchProjects, createProject, uploadDeploy, pollDeploy } from './platform-api.js';
 import { getProjectConfigToSave, loadProjectConfig, saveProjectConfig } from './project-config.js';
+import { MASTRA_PLATFORM_API_URL } from '../auth/client.js';
```

ESLint

[error] `../auth/client.js` import should occur before import of `../auth/credentials.js`

(import/order)